src/js/components/Video.js

```diff
@@ -21,24 +15,23 @@ export default class Video extends Component {
   constructor () {
     super();
 
-    this._onResponsive = this._onResponsive.bind(this);
-    this._onPlaying = this._onPlaying.bind(this);
-    this._onPause = this._onPause.bind(this);
-    this._onEnded = this._onEnded.bind(this);
-    this._onClickControl = this._onClickControl.bind(this);
+    this._hasPlayed = false;
+    this._play = this._play.bind(this);
+    this._pause = this._pause.bind(this);
+    this._togglePlay = this._togglePlay.bind(this);
+    this._toggleMute = this._toggleMute.bind(this);
+    this._seek = this._seek.bind(this);
+    this._mute = this._mute.bind(this);
+    this._unmute = this._unmute.bind(this);
+    this._fullscreen = this._fullscreen.bind(this);
     this._onMouseMove = this._onMouseMove.bind(this);
-    this._onClickChapter = this._onClickChapter.bind(this);
-    this._onFullScreen = this._onFullScreen.bind(this);
 
-    this.state = { playing: false, progress: 0, iconSize: 'large' };
+    this.state = {};
   }
 
-  componentDidMount () {
-    this._responsive = Responsive.start(this._onResponsive);
-    let video = this.refs.video;
-    video.addEventListener('playing', this._onPlaying);
-    video.addEventListener('pause', this._onPause);
-    video.addEventListener('ended', this._onEnded);
+  componentWillMount () {
+    this._update = throttle(this._update.bind(this), 100).bind(this);
```

Comment:
Just for my own knowledge, is there a reason why we want to avoid lodash? Do note that this is just pulling in lodash.throttle and not the entire lodash package. 

I thought about pulling in just the throttle functionality as a common util function in grommet core, but I figured it's better to pull in something that is more battle-tested like lodash.throttle than to come up with my own version or copy and paste from one the many sources out there. Plus, this way, whenever lodash.throttle gets updated, we get that for free.